disnake/interactions/base.py

```diff
@@ -530,6 +532,18 @@ async def send_message(
         InteractionResponded
             This interaction has already been responded to before.
         """
+        if allowed_mentions is not None:
```

Comment:
`allowed_mentions` should never be `None` in the first place if you don't annotate it as Optional